### PR TITLE
[percep] AR tag streaming

### PR DESCRIPTION
**Pull request by qhdwight**
I would be extremely cautious about merging this before being thoroughly tested.

### CODE REVIEW

**Review by niwhsa9 — APPROVED** [review]
LGTM. 

really stupid nit which you should ignore: u can save a cvtcolor operation by swapping channels on the reciever end of the gstreamer pipe which frees up some time on the jetson, but its negligible anyway.

**Comment by cgiger00** [comment]
yes agreed, will test ASAP

File: jetson/percep/main.cpp [hunk 1]
```diff
@@ -130,6 +130,14 @@ int main() {
         DoubleTrack(&detector.getAlvarParams()->maxErroneousBitsInBorderRate).setup("maxErroneousBitsInBorderRate", "ARUCO Debug", 1);
         DoubleTrack(&detector.getAlvarParams()->minOtsuStdDev).setup("minOtsuStdDev", "ARUCO Debug", 20);
         DoubleTrack(&detector.getAlvarParams()->errorCorrectionRate).setup("errorCorrectionRate", "ARUCO Debug", 1);
+
+        std::string const gstLaunch = "appsrc ! video/x-raw, format=BGR !"
+                                    "queue ! videoconvert ! x264enc tune=zerolatency bitrate=1000000 speed-preset=superfast !"
+                                    "rtph264pay ! udpsink host=127.0.0.1 port=5000";
```
Comment (cgiger00) [inline jetson/percep/main.cpp:136]: can you change it to port 5002 or something? in auton 5000 and 5001 are being used by the regular usb cameras

**Review by cgiger00 — CHANGES_REQUESTED** [review]

**Review by cgiger00 — APPROVED** [review]
testing has been good w this

**Review by ankithu — APPROVED** [review]

**Review by tabiosg — APPROVED** [review]
crazy!!!

**Comment by cgiger00** [comment]
fixed with https://github.com/umrover/mrover-workspace/pull/1132

**Pull request closed without merging**